README.md

```diff
@@ -15,11 +15,23 @@ Print the 3MF file, modified to your needs. It's not recommended to edit the mes
 * Infill: 100%
 * Bed temperature: 100°C temp at the first layer to avoid warping.
 
-* Supports: The buckle and rod need to be printed with support. Make sure there is no support on the holder part.
-* Vertical shells:
-  * Buckle: 2 perimeters
-  * Holder and rod: 4 perimeters
-* Horizontal shells: Print all parts with 2 top and 2 bottom horizontal shell thickness. Holder and rod should have 4 top and 4 bottom.
+### Settings Per Part:
+* backing_support:
+  * No support
```

Comment:
Supports are recommended for the backing plate. We used to use two tiny plastic supports baked into the model, but later resigned to using full supports without those.